### PR TITLE
Add nodes that affect speed

### PR DESCRIPTION
**Pull request by realOneplustwo**
New PR because I messed up the old one.
1. Sticky cobwebs will both slow you down and almost nullify your ability to jump.
2. Quicksand will slow you down if you walk on it.
3. Asphalt will increase your speed if you walk on it.

### CODE REVIEW

File: mods/other/physics/init.lua [hunk 1]
```diff
@@ -66,3 +66,10 @@ function physics.remove(name, layer)
 	players[name][layer] = nil
 	update(name)
 end
+
+function physics.layer_exists(name, layer)
+	-- Basic sanity checks uwu
```
Comment (Panquesito7) [inline mods/other/physics/init.lua:71]:
```suggestion
	-- Sanity checks
```

File: mods/other/speed_nodes/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Speed Nodes
+
+Created by oneplustwo.
```
Comment (Panquesito7) [inline mods/other/speed_nodes/README.md:3]:
```suggestion
Created by [realOneplustwo](https://github.com/realOneplustwo).
```

File: mods/other/speed_nodes/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+License: MIT.
```
Comment (Panquesito7) [inline mods/other/speed_nodes/README.md:5]:
```suggestion
## License

Copyright (C) 2021 realOneplustwo

Speed Nodes is licensed under MIT.
See [`LICENSE`](https://github.com/MT-CTF/capturetheflag/blob/master/mods/other/speed_nodes/LICENSE) for more information.
```

File: mods/other/speed_nodes/init.lua [hunk 4]
```diff
@@ -0,0 +1,94 @@
+minetest.register_node("speed_nodes:cobweb", {
```
Comment (Panquesito7) [inline mods/other/speed_nodes/init.lua:1]:
```suggestion
-- Capture the Flag mod: speed_nodes

minetest.register_node("speed_nodes:cobweb", {
```

File: mods/other/speed_nodes/README.md (hunk 3 above)
Comment (realOneplustwo) [reply mods/other/speed_nodes/README.md:5]: Would a relative path be better?

Comment (TSafa-23) [reply mods/other/speed_nodes/README.md:5]: It may be better to just link to the MIT license, also this isn't really necessary

**Comment by realOneplustwo** [comment]
I think this mod would be better used for a specific map instead of adding it to the entire repo, since it just adds nodes.

**Pull request closed without merging**